UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -496,7 +496,23 @@ public enum EventIds
         /// <summary>
         /// 805122 - New access token is added to cache for external end point resource.
         /// </summary>
-        CachingExternalEndPointToken = 805122
+        CachingExternalEndPointToken = 805122,
+        /// <summary>
+        ///  805123 -Preparing to download ENC files based on Product/CellName EditionNumber and UpdateNumber from file share service.
+        /// </summary>
+        FileShareServicePreparingToDownloadENCFilesStart = 805123,
+        /// <summary>
+        ///  805124 -Completed download of ENC files based on Product/CellName EditionNumber and UpdateNumber from file share service.
+        /// </summary>
+        FileShareServicePreparingToDownloadENCFilesCompleted = 805124,
```

Comment:
Is it for preparation or actual download complete?
```suggestion
        FileShareServiceDownloadENCFilesCompleted = 805124,
```

Reply:
Yes updated EventId as per suggested, Thanks!